models/public/midasnet/midasnet.md

```diff
@@ -25,15 +25,15 @@ See [here](https://github.com/intel-isl/MiDaS)
 | Metric            | Value         |
 |-------------------|---------------|
 | Type              | Monodepth     |
-| GFLOPs            | 207.4915      |
-| MParams           | 104.0814      |
+| GFLOPs            | 207.25144     |
+| MParams           | 104.081       |
 | Source framework  | PyTorch\*     |
 
 ## Accuracy
 
-| Metric | Value |
-| ------ | ----- |
-| rmse   | 7.5878|
+| Metric | Value   |
+| ------ | ------- |
+| rmse   | 0.07071 |
```

Comment:
Note that we have recalculated the accuracy of the current model, and it's actually _better_ than this new value. So is the model even worth replacing?

Reply:
This model improves upon the results of the old model in our large-scale evaluation.

Reply:
OK.